openai/validators.py

```diff
@@ -158,17 +158,26 @@ def long_examples_validator(df):
 
     ft_type = infer_task_type(df)
     if ft_type != "open-ended generation":
-        long_examples = df.apply(
-            lambda x: len(x.prompt) + len(x.completion) > 10000, axis=1
-        )
-        long_indexes = df.reset_index().index[long_examples].tolist()
+        def get_long_indexes(d):
+            long_examples = d.apply(
+                lambda x: len(x.prompt) + len(x.completion) > 10000, axis=1
+            )
+            long_indexes = d.index[long_examples].tolist()
```

Comment:
What was the issue with reset_index here?

Reply:
It works either way with the same outcome for this use case; but yes, thinking this over again I'll add that back in.

Reply:
It wasn't making a difference for this particular use case. Leaving it in, then! Sorry, it's been a moment since I had looked over this.